torch/onnx/_internal/exporter.py

```diff
@@ -1277,6 +1277,12 @@ def export(self) -> ONNXProgram:
                     "ONNXScript optimizer is not available. Skipping optimization. "
                     "Please `pip install onnxscript -U` to enable post-export optimization."
                 )
+            except Exception as e:
```

Comment:
We need to make sure the returned model is not changed if optimization fails.

Reply:
In the previous PR, I thought you said it makes copy in optimizer?